src/Build/Construction/Solution/SolutionFile.cs

```diff
@@ -475,8 +482,6 @@ internal void ParseSolutionFile()
             {
                 // Open the file
                 fileStream = File.OpenRead(_solutionFile);
-                // Store the directory of the file as the current directory may change while we are processes the file
-                SolutionFileDirectory = Path.GetDirectoryName(_solutionFile);
```

Comment:
Have you found cases when the current directory changes while processing the solution file? I couldn't find a public-facing way to change directory between where it is current set and where you are now setting it.

Reply:
I have no idea, this comment has been here since the dawn of time.

Reply:
> 2006-04-07 18:38:15
> Initial multiproc code

🤷🏻‍♂️